dojo/jira_link/views.py

```diff
@@ -194,31 +220,30 @@ def check_for_and_create_comment(parsed_json):
     commenter_display_name = comment.get('updateAuthor', {}).get('displayName')
     # example: body['comment']['self'] = "http://www.testjira.com/jira_under_a_path/rest/api/2/issue/666/comment/456843"
     jid = comment.get('self', '').split('/')[-3]
-    jissue = get_object_or_404(JIRA_Issue, jira_id=jid)
-    logging.info(f"Received issue comment for {jissue.jira_key}")
+    try:
+        jissue = JIRA_Issue.objects.get(jira_id=jid)
+    except JIRA_Instance.DoesNotExist:
+        return webhook_responser_handler(404, f"JIRA issue {jid} is not linked to a DefectDojo Finding")
```

Comment:
Should this match the case above and log this as "info"?

```suggestion
        return webhook_responser_handler("info", f"JIRA issue {jid} is not linked to a DefectDojo Finding")
```